beanprice/price.py

```diff
@@ -164,7 +172,7 @@ def parse_single_source(source):
     return PriceSource(module, symbol, bool(invert))
 
 
-def import_source(module_name):
+def import_source(module_name: str) -> Any:
     """Import the source module defined by the given name.
```

Comment:
Having an `Any` return type annotation isn't that useful. Just leave it out.

Reply:
Added SourceModule - it currently just maps to Any, but at least it makes it clearer from the name what we expect.

Reply:
I don't understand why you want to add a type to this at any cost. It is perfectly fine to do not add type information to everything, if there isn't a clear way to define a type.

Reply:
Removing the type for now.

Background: I am coming from strongly typed languages, so I just want to add types everywhere - please let me know when I add too many of them.

In this particular case, there are two reasons why I wanted to try to add more specific type:
1. First, I am not sure what we expect this method returns. Do we expect just a python module, or do we expect python module that has a Source class? If we only expect the python module, which part of the code is expected to check that the module properly implements the Source class? Do we expect to validate it early, or do we expect to get exception in the "leaf" code?
2. I believe there is a bug related to this type - when I used specific type instead of `Any`, there was some code that tried to assign `None` in `PriceSource`, but there was another part of the code that does not check for `None`. I will try to investigate it more in some follow up.